### PR TITLE
vboxmanage-movevm: add page

### PR DESCRIPTION
**Pull request by JimsJam**
- [x] The page(s) are in the correct platform directories: `common`.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/vboxmanage-movevm.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# vboxmanage-movevm
+
+> Move a virtual machine (VM) to a new location on the host system.
+> More information: <https://www.virtualbox.org/manual/ch08.html#vboxmanage-movevm>.
+
+- Move the specified VM:
+
+`VBoxManage movevm {{vm_name}}`
```
Comment (gutjuri) [inline pages/common/vboxmanage-movevm.md:8]: Where does it get moved to?

Comment (JimsJam) [reply pages/common/vboxmanage-movevm.md:8]: Ah my bad will change it!

Comment (JimsJam) [reply pages/common/vboxmanage-movevm.md]: @gutjuri do you know why jiamvy is making the changes and not my account jimsjam?

Comment (kbdharun) [reply pages/common/vboxmanage-movevm.md]: @JimsJam Can you check `git --config user.name` and `git --config email` locally to see if they match your GitHub name & email address?

Comment (JimsJam) [reply pages/common/vboxmanage-movevm.md]: @kbdharun Thank you so much for the help. The e-mail was actually a different one.

File: pages/common/vboxmanage-movevm.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+> Move a virtual machine (VM) to a new location on the host system.
+> More information: <https://www.virtualbox.org/manual/ch08.html#vboxmanage-movevm>.
+
+- Move the specified VM to the current location:
+
+`VBoxManage movevm {{vm_name}}`
+
+- Specify the new location (full or relative pathname) of the VM:
+
+`VBoxManage movevm {{vm_name}} --folder {{path/to/new_location}`
```
Comment (gutjuri) [inline pages/common/vboxmanage-movevm.md:12]:
```suggestion
`VBoxManage movevm {{vm_name}} --folder {{path/to/new_location}}`
```

Thanks :) One more change and this looks good to me

Comment (JimsJam) [reply pages/common/vboxmanage-movevm.md]: Ah my bad, will change it. I will do it more conscientiously on the next page!

File: pages/common/vboxmanage-movevm.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# vboxmanage-movevm
+
+> Move a virtual machine (VM) to a new location on the host system.
+> More information: <https://www.virtualbox.org/manual/ch08.html#vboxmanage-movevm>.
+
+- Move the specified VM to the current location:
```
Comment (kbdharun) [inline pages/common/vboxmanage-movevm.md:6]:
```suggestion
- Move the specified virtual machine to the current location:
```

File: pages/common/vboxmanage-movevm.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+`VBoxManage movevm {{vm_name}}`
+
+- Specify the new location (full or relative pathname) of the VM:
```
Comment (kbdharun) [inline pages/common/vboxmanage-movevm.md:10]:
```suggestion
- Specify the new location (full or relative pathname) of the virtual machine:
```

**Review by kbdharun — APPROVED** [review]
The page description with abbreviation LGTM. However, I would suggest having the command descriptions verbose (descriptive) too. This would make this page consistent with other Virtualbox pages.

**Review by JimsJam — COMMENTED** [review]
Thank you for the feedback, I have changed it now.

**Review by gutjuri — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**